hpi-schul-cloud/nuxt-client
# fix printing qr code

# Short Description

QA note:
Change tested on env:
https://nbc-bc-5526-fix-printing-qr-code.cd.dbildungscloud.dev/
https://brb-bc-5526-fix-printing-qr-code.cd.dbildungscloud.dev/
No issues found. All accepted criteria done: QRcode is show and open. 
See movie in JIRA ticket 





## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-5526


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
